### PR TITLE
feat(ui5-li-custom, ui5-li, ui5-li-groupheader): add accessibleName property

### PR DESCRIPTION
**Pull request by nnaydenow**
Added new accessibleName attribute which allows to have additional custom information of each list item.

Closes: #3014

### CODE REVIEW

**Comment by dobrinyonkov** [comment]
Works fine with screen readers and looks good from accessibility POV.

File: packages/main/src/CustomListItem.js [hunk 1]
```diff
@@ -23,6 +23,18 @@ const metadata = {
 		},
 	},
 	properties: /** @lends sap.ui.webcomponents.main.CustomListItem.prototype */ {
+		/**
+		 * Defines the text alternative of the <code>ui5-li</code>.
```
Comment (fifoosid) [inline packages/main/src/CustomListItem.js:27]: Defines the text alternative of the component

File: packages/main/src/CustomListItem.js [hunk 2]
```diff
@@ -23,6 +23,18 @@ const metadata = {
...
+		 * If not provided a default text alternative will be set, if present.
```
Comment (fifoosid) [inline packages/main/src/CustomListItem.js:28]: <b>Note:</b> If not provided a default text alternative will be set, if present.

File: packages/main/src/GroupHeaderListItem.js [hunk 3]
```diff
@@ -16,6 +16,18 @@ const metadata = {
 	tag: "ui5-li-groupheader",
 	languageAware: true,
 	properties: /** @lends  sap.ui.webcomponents.main.GroupHeaderListItem.prototype */ {
+		/**
+		 * Defines the text alternative of the <code>ui5-li</code>.
```
Comment (fifoosid) [inline packages/main/src/GroupHeaderListItem.js:20]: Same here

File: packages/main/src/StandardListItem.js [hunk 4]
```diff
@@ -85,6 +85,19 @@ const metadata = {
 			defaultValue: ValueState.None,
 		},
 
+		/**
+		 * Defines the text alternative of the <code>ui5-li</code>.
```
Comment (fifoosid) [inline packages/main/src/StandardListItem.js:89]: Same here

**Review by fifoosid — CHANGES_REQUESTED** [review]
Some minor docs to be fixed.

I was wondering why do you add the property to StandardListItem & CustomListItem as they extend ListItem and borrow the property?

**Comment by nnaydenow** [comment]
Hi @fifoosid ,

We add this property to each component because it is not confirmed to other components that extends the ListItem. We are doing this to not affect them.

**Review by fifoosid — APPROVED** [review]

**Pull request merged**